packages/block-editor/src/components/rich-text/index.js

```diff
@@ -493,9 +493,10 @@ export class RichText extends Component {
 			const newColor = computedStyle.color
 				.replace( ')', ', 0.2)' )
 				.replace( 'rgb', 'rgba' );
+			const path = `[contenteditable]:focus ${ boundarySelector }`;
```

Comment:
Is there a reason it not be more specific to a `contenteditable` rendered in `.editor-rich-text` ? While certainly an edge case, would I be wrong in thinking there's still the potential here for some false positives with e.g. `<div contentEditable><RichText /></div>` ?

Reply:
Yeah, we should use the class.